clean_files.txt

```diff
@@ -1 +1,2 @@
 themes/90210/90210.theme.bash
```

Comment:
Maybe change this to the directory name as well? Remove the file name?

Reply:
Yeah good idea plus its an easy win for this theme since it only contains the one file anyways.